app/javascript/react/containers/PodcastIndexContainer.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PodcastIndexTile from '../components/PodcastIndexTile'
+
+class PodcastIndexContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      podcasts: []
+    }
+  }
+
+  componentDidMount() {
+    fetch("/api/v1/podcasts.json")
+      .then((response) => response.json())
+      .then((responseData) => {
+        this.setState({podcasts: responseData})
+      })
+      .catch((error) => {
+        throw error
+      })
```

Comment:
please add the complete set of standard error handling here, per the example in the original fetch reading